src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -97,46 +97,12 @@ export class PartCreationComponent implements OnInit, OnDestroy {
         const gameExists: boolean = await this.joinerService.joinGame(this.partId, this.userName);
         if (gameExists === false) {
             // We will be redirected by the GameWrapper
-            return Promise.resolve();
+            return;
```

Comment:
est-ce que c'est bien équivalent ? (l'objet de retout attendu étant une promise)

Reply:
`async function donneMoiUnChatDesQuilPleut(): Promise<MainCoon> { if (this.ilPleut()) return this.pollux; else return null; } `
pollux est de type MainCoon et pas Promise et ça marche dans les code, async permet de pas devoir préciser que c'est une promise, si je n'm'abuse ! ça vaut pour promise<void> et les retour nulls, je pense!